imagelib/search.php

```diff
@@ -165,35 +165,49 @@
 					?>
 					<div class="flex-form">
 						<div>
-							<label for="imagecount"><?php echo htmlspecialchars($LANG['IMG_COUNT'], HTML_SPECIAL_CHARS_FLAGS) ?>: </label>
-							<select id="imagecount" name="imagecount">
-								<option value="all" <?php echo ($imgLibManager->getImageCount()=='all'?'SELECTED ':''); ?>>All images</option>
-								<option value="taxon" <?php echo ($imgLibManager->getImageCount()=='taxon'?'SELECTED ':''); ?>>One per taxon</option>
-								<?php
-								if($specArr){
-									?>
-									<option value="specimen" <?php echo ($imgLibManager->getImageCount()=='specimen'?'SELECTED ':''); ?>>One per specimen</option>
-									<?php
-								}
+							<div>
+								<label for="imagecount"><?php echo htmlspecialchars($LANG['IMG_COUNT'], HTML_SPECIAL_CHARS_FLAGS) ?>: </label>
+							</div>
+							
+							<input style="margin-top: 0.5rem" id="countAll" type="radio" name="imagecount" value="0" CHECKED><label for="countAll"> All Images</label> <br/>
```

Comment:
Maybe change this to just "All" instead of "All Images" so that there's no conflict (see attached Total Validator Pro run).

Reply:
Also, since we're changing text, let's go ahead and internationalize it.